pkg/backup/remap_crd_version_action.go

```diff
@@ -68,7 +71,26 @@ func (a *RemapCRDVersionAction) Execute(item runtime.Unstructured, backup *v1.Ba
 		return item, nil, nil
 	}
 
-	// We've got a v1 CRD, so proceed.
+	// This plugin will exit if the CRD was installed via v1beta1 but the cluster does not support v1beta1 CRD
+	supportv1b1 := false
+CheckVersion:
+	for _, g := range a.discoveryHelper.APIGroups() {
+		if g.Name == apiextv1.GroupName {
+			for _, v := range g.Versions {
+				if v.Version == apiextv1beta1.SchemeGroupVersion.Version {
+					supportv1b1 = true
+					break CheckVersion
+				}
+			}
+
+		}
+	}
+	if !supportv1b1 {
+		a.logger.Info("Exiting RemapCRDVersionAction, the cluster does not support v1beta1 CRD")
+		return item, nil, nil
+	}
```

Comment:
Can we write it something like
```go
gvr, _, err := a.discoveryHelper.ResourceFor(
	schema.GroupVersionResource{
		Group:    "apiextensions.k8s.io",
		Version:  "v1beta1",
		Resource: "customresourcedefinitions",
	})
```

Reply:
@jenting Thanks for the comment, but this seems not working.  It returns a `no matching` error on cluster that supports `v1beta1` CRD
seems the helper will only put the preferred version in the map unless the `APIGroupVersionsFeatureFlag` is set:
https://github.com/vmware-tanzu/velero/blob/15136745488c07201b13a8ebd738eb6e27fffaf1/pkg/discovery/helper.go#L161

Reply:
I see, make sense.